development/tools/jobs/tester/componentsuite.go

```diff
@@ -57,13 +57,13 @@ func (s ComponentSuite) Run(t *testing.T) {
 func (s ComponentSuite) preMasterTest(jobConfig config.JobConfig) func(t *testing.T) {
 	return func(t *testing.T) {
 		job := FindPresubmitJobByNameAndBranch(
-			jobConfig.Presubmits[s.repositorySectionKey()],
+			jobConfig.PresubmitsStatic[s.repositorySectionKey()],
```

Comment:
I don't know why we have to change the naming as part of the go mod switch. Unless I'm missing something. Same for `Postsubmits` -> `PostsubmitsStatic`

Reply:
Nvm, I see why the change is necessary.